src/GraphQL.SystemTextJson/ExecutionResultJsonConverter.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GraphQL.SystemTextJson
+{
+    public class ExecutionResultJsonConverter : JsonConverter<ExecutionResult>
```

Comment:
I am pleasantly surprised how this implementation looks like existing one.

Reply:
Yea it's pretty lucky. The only thing I was worried about was that I'm not currently passing the `options` recursively down, so if someone wanted PascalCase all the way down, there could be a gap there. The thing is, you're not supposed to pass it down, so you have to `new` up a new options for each recursion which is just nasty. 

Since all tests currently pass, until a test is written that breaks it, or someone complains, kinda tempted to just leave as is as an MVP.

Reply:
ok